Bigdata-flink-example/Flink-cdc-example/pom.xml

```diff
@@ -147,7 +147,7 @@
             <groupId>org.scala-lang</groupId>
             <artifactId>scala-library</artifactId>
             <version>2.12.12</version>
-            <scope>compile</scope>
```

Comment:
What is the impact of this change?

Reply:
We don't need to break into Scala dependency